docs/design/lexical_conventions/symbolic_tokens.md

```diff
@@ -62,67 +63,41 @@ overloading.
-
-This list is expected to grow over time as more symbolic tokens are required by
-language proposals.
-
-Note: The above list only covers up to
-[#601](https://github.com/carbon-language/carbon-lang/pull/601) and more have
-been added since that are not reflected here.
-
-### Whitespace
-
-Carbon's rule for whitespace around operators have been designed to allow the
-same symbolic token to be used as a prefix operator, infix operator, and postfix
-operator in some cases. To make parsing operators unambiguous, we require
-whitespace to be present or absent around the operator to indicate its fixity,
-with binary operators having whitespace on both sides, and unary operators
-lacking whitespace between the operator and its operand. However, there are some
-cases where omitting whitespace around a binary operator can aid readability,
-such as in expressions like `2*x*x + 3*x + 1`. In such cases, the operator with
-whitespace on neither side is treated as binary if the token immediately before
-the operator indicates the end of an operand and the token immediately after
-indicates the beginning of an operand.
-
-Identifiers, literals, and brackets of any kind, facing away from the operator,
-are defined as tokens that indicate the beginning or end of an operand. For
-error recovery purposes, no expression context can be preceded by a token that
-looks like the end of an operand, and no expression context can be followed by a
-token that looks like the start of an operand, except in function definitions
-where `{}` is the body of the function.
-
-From the perspective of token formation, there are four variants of each
-symbolic token: a binary variant with whitespace on both sides, a binary variant
-with whitespace on neither side, a unary variant with whitespace on neither
-side, and prefix and postfix variants with whitespace on the left and right
-sides, respectively. In non-operator contexts, any variant of a symbolic token
-is acceptable, but in operator contexts, only the appropriate variant can be
-used.
-
-The whitespace rule was designed to strike a balance between simplicity and
-expressiveness for the programmer, and simplicity and good support for error
-recovery in the implementation. The rule's allowance for omitting whitespace
-around binary operators aids readability, but it can cause errors if not used
-carefully, particularly in function definitions. Despite this, the rule provides
-the necessary cues for human readers to understand the code, while still
-allowing for unambiguous parsing of operators.
+| Symbolic Tokens and Explanation                                                                                |
+| -------------------------------------------------------------------------------------------------------------- | ------------ |
+| `+` Addition                                                                                                   |
```

Comment:
It looks like the `|` separators in this table are missing. This might be related to my comment below on "Bitwise OR", so probably fix that first.